### ISSUE TITLE
infinite scrolling loop

### ISSUE THREAD

**Issue by steltenpower** [jupyterlite/jupyterlite#1452]
## Description

Sometimes scrolling upwards through my notebook, just before it reaches the top jumps down several pages again. Scrolling by swiping my touchpad with 2 fingers.

## Reproduce

No pattern found yet

## Context

JupyterLite Version 0.3.0
Windows 10 Enterprise 22H2
[Version 1.67.134 Chromium: 126.0.6478.186 (Official Build) (64-bit)](https://brave.com/latest/)

**Comment by Carreau** [comment]
Is that only Jupyterlite ? And is it in iframes or in full pages. I have issues seeing how this could  be specific to jupyterlite.

**Comment by steltenpower** [comment]
It is on https://jupyterlite.github.io/demo/lab/index.html hoping that answers your question about no iframes or full page.
This is the first time I have ever seen this behaviour on any software or any device, ever. In the past I've used JuPyteR quite a bit, never happened there. I just tried https://jupyter.org/try-jupyter/notebooks/?path=notebooks/Intro.ipynb and couldn't replicate the behaviour there either.
